woocommerce/woocommerce-paypal-payments
# Send transaction ids to woocommerce

**Issue**: [PCP-65](https://inpsyde.atlassian.net/browse/PCP-65).

---

### Description

Add transaction URL to the WC Order. Make it clickable so the site admin could check transaction on the PayPal site.
***
Add the `_transaction_id` field to a new WooCommerce order so it will be displayed on the edit order page. Add `view_transaction_url` to the payment gateways so the displayed transaction could be clickable. For each order, the payment mode is saved to the meta field. Depending on that, each order will be displayed sandbox or live transaction URL.

_Note:_ Transaction URL displayed for the orders only.

### Steps to test:


1. Purchase any product using PayPal gateway.
2. Open the order page in the WC admin and check the line under the order number (where the transaction id is usually displayed). It should contain clickable transaction id.
3. Login into your test merchant PayPal account in a new tab.
4. Return to the tab with the WC order and click transaction id. You should see the PayPal page with the transaction.
5. Do the same procedure using Credit Card payment.

_Note:_ If you will click on the transaction number and login into PayPal account later you will not be redirected to the transaction page. So, you need to be logged in first.

### Documentation




### Changelog entry
> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.

* Add - Display clickable transaction id on the edit order page.
